parser/parser/codegen/generate_diagnostics.cc

```diff
@@ -6,6 +6,8 @@ using namespace std;
 namespace {
 
 tuple<string, string> MESSAGES[] = {
+    {"InternalError", "internal error while parsing: {}"},
```

Comment:
(non-blocking) Is it worth making this particular error translate to an actual internal error from `core/`?

Reply:
Good point, will do.